### PR TITLE
[opencc] De-vendor rapidjson

### PR DESCRIPTION
**Pull request by Adela0814**
For https://github.com/microsoft/vcpkg/issues/26179

Add new feature `rapidjson`.

Tested all features in the following triplets:

- x64-windows
- x86-windows
- x64-windows-static

### CODE REVIEW

**Review by BillyONeal — CHANGES_REQUESTED** [review]
This can't be a feature, it must be unconditional. (Dependencies must not be vendored because vendoring stuff creates link incompatibilities)

**Comment by BillyONeal** [comment]
(Thank yo8u so much for trying to fix this though!)

File: ports/opencc/portfile.cmake [hunk 1]
```diff
@@ -12,9 +14,10 @@ vcpkg_add_to_path("${PYTHON3_DIR}")
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    OPTIONS
+    OPTIONS ${FEATURE_OPTIONS}
```
Comment (Thomas1664) [inline ports/opencc/portfile.cmake:17]: Where is `vcpkg_check_features`?

Comment (Adela0814) [reply ports/opencc/portfile.cmake]: Removed.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**